components/collective-page/CollectiveNotificationBar.js

```diff
@@ -98,6 +98,9 @@ const messages = defineMessages({
 });
 
 const getNotification = (intl, status, collective, host, LoggedInUser, refetch) => {
+  const numberOfAdmins = [CollectiveType.EVENT, CollectiveType.PROJECT].includes(collective.type)
```

Comment:
I think checking for `parentCollective` should be enough.
```suggestion
  const numberOfAdmins = collective.parentCollective
```